src/log_level.rs

```diff
@@ -37,19 +52,6 @@ impl From<LogLevel> for &'static str {
     }
 }
 
-impl std::str::FromStr for LogLevel {
-    type Err = &'static str;
-
-    fn from_str(s: &str) -> Result<LogLevel, Self::Err> {
-        match s {
-            "quiet" => Ok(Self::Quiet),
-            "info" | "all" => Ok(Self::Info),
```

Comment:
I wonder if we can coerce "all" into "info" without breaking this

Reply:
https://docs.rs/clap/latest/clap/_derive/index.html#possible-value-attributes

maybe this?

```rust
enum LogLevel {
  #[value(alias("all"))]
  Info
}
```